commands/meme_stats.py

```diff
@@ -27,26 +27,29 @@
 @lightbulb.command(
     "memestats", "See the aggregated server or someone's average meme rating over time"
 )
-@lightbulb.implements(lightbulb.PrefixCommand, lightbulb.SlashCommand)
-async def main(ctx: lightbulb.Context):
+@lightbulb.implements(
+    lightbulb.PrefixCommand, lightbulb.SlashCommand, lightbulb.UserCommand
+)
+async def main(ctx: lightbulb.Context | lightbulb.UserContext):
```

Comment:
This is an odd type signature, since UserContext inherits from Context. Can it just be of type Context?

Reply:
Both are different. If used as slash command, `ctx.options` has more properties.
Where as if used as user menu command, `ctx.options` has only `target` property.